bot/database.py

```diff
@@ -96,8 +116,19 @@ async def change_password(self, nickname: str, password: str, new_password: str,
         await worksheet.update_cell(cell.row, cell.col + 1, hashed_password)
         return True
     
-    async def increment_counter(self) -> int:
-        worksheet = await self._get_worksheet(1)
-        cell = await worksheet.cell(1, 2)
-        await worksheet.update_cell(1, 2, int(cell.value) + 1)
-        return int(cell.value)
\ No newline at end of file
+    async def login(self, nickname: str, password: str, userid: str) -> bool:
+        valid = await self.is_password_and_nickname_valid(nickname, password, userid)
+        if valid:
+            self.logged_in_users[userid] = nickname
```

Comment:
I think it would be better security-wise if we hash the userid, wdyt?

Reply:
Mas readable din `is_valid`

Reply:
sige sige add ko waitteuu